Functions/xASL_init_SubStructs.m

```diff
@@ -0,0 +1,47 @@
+function [x] = xASL_init_SubStructs(x)
+%ExploreASL_init_SubStructs Initialize ExploreASL x structure substructs.
+%
+% FORMAT: [x] = ExploreASL_init_SubStructs(x)
+%
+% INPUT:
+%   x       - ExploreASL x struct (STRUCT, REQUIRED)
+%
+% OUTPUT:
+%   x       - ExploreASL x struct
+%
+% -----------------------------------------------------------------------------------------------------------------------------------------------------
+% DESCRIPTION:  Initialize the ExploreASL x structure substructs/fields.
+%               Only fields which do not exist so far are added.
+%               This script is supposed to help with the overall modularity of ExploreASL.
+%               This script is identical to the function ExploreASL_Initialize_SubStructs within
```

Comment:
> Wouldn't create two scripts that are supposed to be identical. Instead, I would then remove the subfunction from ExploreASL_Initialize, and if needed initialize the path at this point. Keep it clean and simple ;)

Agree. Let's improve this later on though, since it's only an optimization and not required for the revamp. I don't like changing the order of the ExploreASL_Initialize code that randomly. We have to be careful here. Maybe we can move an xASL_init_SubStructs call to the part after the addpath in ExploreASL_Initialize and remove the subfunction there, but then we have to be sure that all the lines & subfunctions before have all the substructs they need.